content/logs/faq/log-collection-troubleshooting-guide.md

```diff
@@ -52,11 +52,36 @@ The Datadog Agent only collects logs that have been written after it has started
 
 The `datadog-agent` does not run as root (and we do not recommend that you make it run as root, as a general best-practice). For this reason, when you configure your `datadog-agent` to tail log files (for custom logs or for integrations) you need to take special care to ensure the `datadog-agent` user has read access to tail the log files you want to collect from.
 
-If the `datadog-agent` user does not have read access to the files you configure it to tail, then a permissions error is captured in the `/var/log/datadog/agent.log`. The Agent requires to have an execute permission on the directory containing the files (this is the default permission required to be able to list the file contained in a directory).
+Otherwise there should be a similar message in the Agent `status`:
+
+{{< img src="logs/agent-log-executable-permission-issue.png" alt="Permission issue" responsive="true" style="width:70%;">}}
+
+In those cases it is possible to run the `namei` command to have more information about the file permission:
+
+```
+> namei -m /var/log/application/error.log
+> f: /var/log/application/error.log
+ drwxr-xr-x /
+ drwxr-xr-x var
+ drwxrwxr-x log
+ drw-r--r-- application
+ -rw-r----- error.log
+```
+
+Here the `application` directory is not executable therefore the Agent cannot list its files. And the Agent do not have read permission on the file `error.log`.
+Adding those two permissions thanks to the [chmod command][7] solves the issue.
+
+{{< img src="logs/agent-log-agent-log-permission-ok.png" alt="Permission OK" responsive="true" style="width:70%;">}}
 
 **Note**: When adding the appropriate read permissions, make sure that these permissions are correctly set in your log rotation configuration. Otherwise, on the next log rotate, the Datadog Agent may lose its read permissions.
 Set permissions as `644` in the log rotation configuration to make sure the Agent has read access to the files.
 
+## Permission issue and Journald
+
+When collecting logs from journald, make sure that the Datadog Agent user is added in the systemd group as shown in the [journald integration][8].
+
+The journald API does not return a permission denied error but an empty payload so no explicit message can be raised in the Agent status or logs if the permission is not set.
```

Comment:
I'm having a tough time parsing this sentence. Are you saying that if the perms are incorrect, then journald will just send an empty payload (instead of something useful, i.e. an explicit message)?

Reply:
Exactly. 

Therefore it is not possible for us to know if there is just no logs or if we don't have the permission to see the data.

Reply:
```suggestion
Note that journald will send an empty payload if the file permissions are incorrect. Accordingly, it is not possible to raise or send an explicit error message in this case.
```